docs/UserGuide.md

```diff
@@ -1,4 +1,4 @@
----
+    ---
```

Comment:
This addition seems to significantly change the layout of the user guide. Could you confirm that it's intended?

Reply:
i didnt realise that i did this... can delete ah.